guides/common/modules/con_lifecycle-status-of-rhel-hosts.adoc

```diff
@@ -0,0 +1,9 @@
+[id="lifecycle_status_of_rhel_hosts_{context}"]
+= Lifecycle Status of {RHEL} Hosts
+
+{Project} provides multiple mechanisms to display information about upcoming End of Support (EOS) events for your {RHEL} clients.
+
+When either the end of maintenance support or the end of extended lifecycle support approaches in a year, you will see a notification banner in the {ProjectWebUI} if you have hosts with that {EL} version.
+The notification provides information about the {EL} version, the number of hosts running that version in your environment, the lifecycle support, and the expiration date.
+Along with other information, the {EL} lifecycle column is visible in the report.
```

Comment:
```suggestion
Along with other information, the {EL} lifecycle column is visible in the report.
```
What report?

Reply:
That should be "notification". Changing from "report" to "notification".